x-pack/plugins/security_solution/common/detection_engine/utils.ts

```diff
@@ -93,3 +93,12 @@ export const isSuppressionRuleConfiguredWithMissingFields = (ruleType: Type) =>
 export const isSuppressionRuleInGA = (ruleType: Type): boolean => {
   return isSuppressibleAlertRule(ruleType) && SUPPRESSIBLE_ALERT_RULES_GA.includes(ruleType);
 };
+
+export const shouldShowResponseActions = (ruleType: Type | undefined, featureFlag: boolean) => {
```

Comment:
`featureFlag: boolean` is too broad, i'd suggest to use the name of the feature flag

Reply:
Thanks @gergoabraham , applied your suggestions. 
The tests will not be failing, since there's just one UI test that contains the feature flag set to true. The other tests don't really care if we have the feature flag or not.

But I'll double check tomorrow if there's something missing, thanks!